devtools/create_project/msvc9/create_project.vcproj

```diff
@@ -82,6 +82,9 @@
 			/>
 			<Tool
 				Name="VCPostBuildEventTool"
+				CommandLine="@echo off
+xcopy /Y &quot;$(TargetPath)&quot; &quot;$(SolutionDir)\..\..\..\dists\msvc\&quot;
+xcopy /Y &quot;$(TargetPath)&quot; &quot;$(SolutionDir)\..\..\..\dists\codeblocks\&quot;
```

Comment:
Missing terminating quote, and VS2008 will run this all as one line. This monstrosity (generated by VS2008) works:

`CommandLine="@echo off&#x0D;&#x0A;xcopy /Y &quot;$(TargetPath)&quot; &quot;$(SolutionDir)\..\..\..\dists\msvc\&quot;&#x0D;&#x0A;xcopy /Y &quot;$(TargetPath)&quot; &quot;$(SolutionDir)\..\..\..\dists\codeblocks\&quot;"`